x/programs/rust/wasmlanche-sdk/src/state.rs

```diff
@@ -169,13 +235,20 @@ impl<'a, K: Key> State<'a, K> {
     }
 }
 
-fn get_bytes(key: &[u8]) -> Option<Vec<u8>> {
+fn get_bytes(key: &[u8]) -> Option<CacheValue> {
     #[link(wasm_import_module = "state")]
     extern "C" {
         #[link_name = "get"]
         fn get_bytes(ptr: *const u8, len: usize) -> HostPtr;
     }
 
+    #[derive(BorshSerialize)]
+    struct GetArgs<'a> {
+        key: &'a [u8],
+    }
```

Comment:
Go side needs to change before I can remove this. 

Did you want to do this @dboehm-avalabs?